src/components/announcements-list/AnnouncementsList.tsx

```diff
@@ -0,0 +1,60 @@
+import { Row } from 'antd';
+import React from 'react';
+import { AnnouncementCard } from '../AnnouncementCard';
+import styled from 'styled-components';
+import axios, { AxiosInstance } from 'axios';
+
+const COLUMNS_PER_ROW = 3
+const NO_ANNOUNCEMENTS_MESSAGE = "There are currently no announcements!"
+const ANNOUNCEMENTS = '/api/v1/announcements/'
+const AuthAxiosInstance: AxiosInstance = axios.create({
+    baseURL: process.env.REACT_APP_API_DOMAIN,
+    timeout: 10000,
+    headers: {
+        'Content-Type': 'application/json',
+    },
+});
```

Comment:
You shouldn't be defining a new instance here, you should use the existing one.

Reply:
Good point. I'll make the one that's already defined exportable.